### ISSUE TITLE
enforce newline for single-statement if

### ISSUE THREAD

**Issue by devsnek** [eslint/eslint#9604]
**Please describe what the rule should do:**
enforce putting a newline for a single-statement if
```js
if (condition)
  fn();
```
instead of
```js
if (condition) fn();
```

**What category of rule is this? (place an "X" next to just one item)**
[x] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
if (condition) fn();
```

**Why should this rule be included in ESLint (instead of a plugin)?**
its an extremely common pattern (mostly in larger codebases).
i'm actually surprised it wasn't already a implemented.

**Comment by not-an-aardvark** [comment]
Does [`nonblock-statement-body-position`](https://eslint.org/docs/rules/nonblock-statement-body-position) do what you're looking for?

**Comment by devsnek** [comment]
ohey, thanks. I'm suprised my googling didn't find that.
